Source/WebCore/page/CaptionUserPreferencesMediaAF.cpp

```diff
@@ -346,8 +346,7 @@ String CaptionUserPreferencesMediaAF::captionsWindowCSS() const
 
 String CaptionUserPreferencesMediaAF::captionsBackgroundCSS() const
 {
-    // This default value must be the same as the one specified in mediaControls.css for -webkit-media-text-track-past-nodes
-    // and webkit-media-text-track-future-nodes.
+    // FIXME: This is potentially wrong since modern media controls happened.
```

Comment:
This comment is vague. Nobody will remember the name "modern media controls" in the future. Likely would be better to quote the year we changed the media controls implementation on most platforms or something like that. Put yourself in the shoes of the future person reading the comment.

But also, maybe you could quickly try to assess if this is wrong or not, instead of just reflecting that it’s no longer relevant whether it matches an unused CSS source file?

Reply:
Thanks for the nudge. This was indeed fairly easy to confirm to still be correct. We're not quite rid of the "modern media controls" moniker just yet, but if we wanted to globally rename to "media controls" at some future point once this has had some bake time we could.

Reply:
I actually considered suggesting renaming the CSS files while I was reviewing this. We could surely do the rename today; it's just a matter of how much time and effort do you want to spend on not super valuable work. Probably it would be quick, hopefully....